README.md

```diff
@@ -16,6 +16,10 @@ A detailed list of prerequisites can be found in [`learners/prereqs.md`](learner
 
 - Make a suggestion or correct an error by [raising an Issue](https://github.com/datacarpentry/image-processing/issues).
 
+Please see the [CONTRIBUTING.md](CONTRIBUTING.md) for contributing guidelines and details on how to get involved with 
```

Comment:
```suggestion
Please see [CONTRIBUTING.md](CONTRIBUTING.md) for contributing guidelines and details on how to get involved with 
```
We say 'the README' but I'm not sure about writing 'the CONTRIBUTING.md' --- to be extra safe I would write 'the CONTRIBUTING.md file!'